spec/rails_helper.rb

```diff
@@ -108,3 +108,6 @@
     with.library :rails
   end
 end
+
+# Set the locale for Faker gem to en-GB
+Faker::Config.locale = 'en-GB'
```

Comment:
a comment why might be nice :-)

Reply:
I debated adding one but didn't in the end! I'll add one now..

Reply:
Fixed